yarn.lock

```diff
@@ -10443,7 +10425,7 @@ typescript@^3.0.3:
   resolved "https://registry.yarnpkg.com/typescript/-/typescript-3.9.7.tgz#98d600a5ebdc38f40cb277522f12dc800e9e25fa"
   integrity sha512-BLbiRkiBzAwsjut4x/dsibSTB6yWpwT5qWmC2OfuCg3GgVQCSgMs4vEctYPhsaGtd0AeuuHMkjZ2h2WG8MSzRw==
 
-typescript@^4.0.5:
+typescript@^4.0.5, typescript@~4.0.0:
```

Comment:
This seems odd, but not worth the trouble to investigate, I think.

Reply:
It doesn't seem odd to me.  Can you please explain?